docs/releases.rst

```diff
@@ -0,0 +1,10 @@
+Releases
+========
+
+This page details the changes in the various ``django-formtools`` releases.
+
+1.0
+---
+
+- Added the ``request`` parameter to :meth:`WizardView.get_prefix
```

Comment:
Separate... just mentioned it in passing and thought it'd be easy enough to add if you are committing this now.